tools/arxiv_import/import_pipeline_multigpu.py

```
@@ -0,0 +1,892 @@
+                        f"{embedding_count} embeddings"
+                    )
+
+                    if batches_stored % 10 == 0:
+                        print(f"Storage: {batches_stored} batches stored ({stats_dict['total_created']} papers)")
+
+                except Empty:
+                    if stop_event.is_set() and output_queue.empty():
+                        break
+                    continue
+                except Exception:
+                    logger.exception(f"Storage error")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)